pkg/cmd/pipeline/start.go

```diff
@@ -39,13 +43,22 @@ const (
 )
 
 type startOptions struct {
+	params             cli.Params
```

Comment:
better to rename it something else? it's confusing with pipelinerun `param`

Reply:
I have addressed this and updated the PR